News/configReader.py

```diff
@@ -11,16 +11,18 @@
 class ConfigurationReader():
     __APIKEY=None
     __WebsiteSupported=[]
+    __TopicsSupported=[]
     __Limit=None
     def __init__(self):
-        with open("config.yml", 'r') as ymlfile:
+        with open("/Users/Stormy/PycharmProjects/News-At-Command-Line/config.yml", 'r') as ymlfile:

```

Comment:
This is not right. This path will not hold valid for every user. Thus not correct. The earlier one will search for config.yml in the present directory and make sure it is the relative path and not the full path..